.ci.yaml

```diff
@@ -2139,6 +2139,7 @@ targets:
 
   - name: Linux_build_test flutter_gallery__transition_perf
     recipe: devicelab/devicelab_drone_build_test
+    bringup: true # https://github.com/flutter/flutter/issues/1258979
```

Comment:
Nit: This issue is returning a 404, is it supposed to be https://github.com/flutter/flutter/issues/125897 ?

Reply:
Actually did both, after it starts passing I'll remove the bringup.